### PR TITLE
config: Use toml to store repo info. Bug fixes.

### LINKED ISSUE #6: Take repo urls from a config file.

### PR DESCRIPTION
**Pull request by deekoder**
Fixes #6. Part fix for #2 (provides the data for client to use the hex in css)

### CODE REVIEW

File: buggy.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*
+ * Buggy, (C) 2016,2017 Minio, Inc.
```
Comment (harshavardhana) [inline buggy.go]: This should be just 2017, since its a new file - perhaps call this `main.go` as filename.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: getPRs.go [hunk 2]
```diff
@@ -424,6 +413,12 @@ func populatePRs(url string) {
 
 func getPRs(w http.ResponseWriter, req *http.Request) {
 	pRequests = nil
+	// One Minio.
+	for _, rName := range config.RepoNames {
+		populateIssues(`https://api.github.com/repos/` + rName + `/pulls?state=open&per_page=100&access_token=`)
+		fmt.Printf("Repo: %s \n", rName)
```
Comment (harshavardhana) [inline getPRs.go]: is this printing necessary?

File: gitIssues.go [hunk 3]
```diff
@@ -249,16 +225,10 @@ func getIssues(w http.ResponseWriter, req *http.Request) {
...
-	populateIssues(`https://api.github.com/repos/minio/minio-js/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/minio-java/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/minio-py/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/doctor/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/bosh-release/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/minfs/issues?state=open&per_page=100&access_token=`)
-	populateIssues(`https://api.github.com/repos/minio/minio-dotnet/issues?state=open&per_page=100&access_token=`)
+	for _, rName := range config.RepoNames {
+		populateIssues(`https://api.github.com/repos/` + rName + `/issues?state=open&per_page=100&access_token=`)
+		fmt.Printf("Repo: %s \n", rName)
```
Comment (harshavardhana) [inline gitIssues.go]: Is this printing necessary, is it just for debugging?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by deekoder** [comment]
Done @harshavardhana. Incorporated all the feedback.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**